etrago/cluster/networkclustering.py

```diff
@@ -1409,9 +1471,13 @@ def kmedoids_dijkstra_clustering(etrago):
 def run_spatial_clustering(self):
 
     if self.args["network_clustering"]["active"]:
+        
+        if self.args["network_clustering_ehv"]:    
+        
+            self.adapt_crossborder_buses()
 
         self.network.generators.control = "PV"
-
+        
```

Comment:
It looks like you should run `black` again...

Reply:
Black was used again.